.github/workflows/dev-deploy.yml

```
@@ -0,0 +1,111 @@
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Checkout dev
+      uses: actions/checkout@v4
+  
+    - name: Connect to EC2 and install docker 
+      env:
+        EC2_HOST: ${{ secrets.EC2_HOST }}
+        EC2_USER: ${{ secrets.EC2_USERNAME }}
+        EC2_SSH_KEY: ${{ secrets.SSH_PRIVATE_KEY }}
+      run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:3:48: Double quote to prevent globbing and word splitting

(shellcheck)